client/src/components/BookTripCard/BookTripCard.css

```diff
@@ -0,0 +1,22 @@
+.Card img {
+  width: 45%;
+  height: 100%;
```

Comment:
You may need to change from percentages to rem. Check in with the responsive team about all percentages used in your CSS.

Reply:
this is for a different component on a seperate branch...